lib/usd/pxrUsdPreviewSurface/plugInfo.json

```diff
@@ -4,9 +4,9 @@
             "Info": {
                 "UsdMaya": {
                     "PrimWriter": {
-                        "mayaPlugin": "pxrUsdPreviewSurface",
+                        "mayaPlugin": "mayaUsdPlugin",
```

Comment:
What was the reason for switching from preview surface plugin to mayaUsdPlugin?

Reply:
FWIW. The core library should be independent of any of plugins and I wouldn't want us to break this rule.

Reply:
plugInfo.json will be removed.